src/pages/Feeds/Feeds.js

```diff
@@ -1,12 +1,29 @@
 import React from 'react';
-import WriteFeed from '../../component/WriteFeed/WriteFeed';
+import Feed from './Feed/Feed';
 
-const Feeds = () => {
+const Feeds = ({ feedData }) => {
   return (
-    <div>
-      Feeds
-      <WriteFeed />
-    </div>
+    <>
+      {feedData &&
+        feedData.map(ele => {
+          return (
+            <Feed
+              key={ele.id}
+              id={ele.id}
+              feeduserId={ele.username}
+              feeduserImg="./images/img.jpg"
+              content={ele.content}
+              alt={ele.username}
+              designType={ele.design_type}
+              src={ele.image_url}
+              postedDate={ele.date}
+              tags={ele.tag}
+              comment={ele.comment}
+            />
```

Comment:
```suggestion
            <Feed
              key={ele.id}
              ele={ele}
            />
```
이런식으로 ele 자체를 넘겨주고, 받아서 쓰는 자식 컴포넌트에서 구조분해 할당을 통해 사용해주세요 :)

Reply:
참고로 변수명도 ele 보단 더 직관적으로 지어주세요!

Reply:
넵 ㅎㅎ